installer/charts/rhtap-dh/templates/plugins-content.yaml

```diff
@@ -143,16 +143,23 @@ plugins:
   # RBAC
   #
   {{- $githubSecretObj := (lookup "v1" "Secret" $integrationNamespace "rhtap-github-integration") }}
-  {{- if and .Values.developerHub.RBAC.enabled $githubSecretObj }}
+  {{- $gitlabSecretObj := (lookup "v1" "Secret" $integrationNamespace "rhtap-gitlab-integration") }}
```

Comment:
Clean-up: maybe you can move those var definitions to the top of the file, and use them in the CI section as well?